dp-inntekt-api/src/test/kotlin/no/nav/dagpenger/inntekt/rpc/GrpcTest.kt

```diff
@@ -0,0 +1,64 @@
+package no.nav.dagpenger.inntekt.rpc
+
+import io.grpc.BindableService
+import io.grpc.ManagedChannel
+import io.grpc.ServerBuilder
+import io.grpc.inprocess.InProcessChannelBuilder
+import io.grpc.inprocess.InProcessServerBuilder
+import io.grpc.testing.GrpcCleanupRule
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.Executors
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
```

Comment:
junit4?

Reply:
Ja :( 

Den https://github.com/navikt/dp-inntekt-api/blob/cd8be062d72af689c7ad3db19efea2392d1b41c3/dp-inntekt-api/src/test/kotlin/no/nav/dagpenger/inntekt/rpc/GrpcTest.kt#L21 er ganske innfløkt og det fantes ikke noe alternativ for Junit 5